### DISCUSSION TITLE
USA Mass Purchase

### DISCUSSION THREAD

**Discussion by LordNex** [justLV/onju-voice#21] [General]
So I thought I'd start a discussion to see how many people would want to put in an order from the USA. I know I need 4-5 myself so I figured we might be able to get a better cost if we pool our resources. Ideas?

**Comment by bkabela** [comment]
I'm located in Iowa and I'd be interested in 3.

**Reply by LordNex** [reply]
I'm about to put in the order. Are you still wanting 3?

I'm not trying to make money off of these yet so the total cost for 10 was almost right at $300 including shipping to me. That will make each board approximately $30-31 each not including what it cost to send to you. If you would like to talk over the phone or whatever we can figure out the cheapest way to get them shipped. I'm in Illinois so it'll probably be close to the same for everyone. Once I can see the packing and over all size. I'll have a better idea on how to ship it. I'll probably just go USPS Boxed Standard unless you guys request another form. Let me know as I'll only be selling to you 2, if you still want them.

**Comment by qnlbnsl** [comment]
I am located in Tampa Florida, and I would be interested in 3 as well.

**Reply by LordNex** [reply]
Hey I'm getting ready to order. Do you still want 3? I'm not trying to make money off of these yet so the total cost for 10 was almost right at $300 including shipping to me. That will make each board approximately $30-31 each not including what it cost to send to you. If you would like to talk over the phone or whatever we can figure out the cheapest way to get them shipped. I'm in Illinois so it'll probably be close to the same for everyone. Once I can see the packing and over all size. I'll have a better idea on how to ship it. I'll probably just go USPS Boxed Standard unless you guys request another form. Let me know as I'll only be selling to you 2, if you still want them.

**Comment by LordNex** [comment]
Ok well I'm hoping to get a team working on Zigbee Devices to look at this and see if we can add that (thread/matter/zigbee) support on these boards. There are some newer ESP chipsets coming out with faster processors and different hardware. 

Personally I use Zigbee a lot and am hoping matter/thread really comes in later to give us more options. Either way most of these ESP chipsets can come with a SoC Zigbee module. 

The other issue is we have to order in chucks of 5. I'm guessing that's how many fit on one die so they o ly well in those quantities. I can order 10 but one of us will be out. If I order 15 we're going to have way more than we need. So we'll need to figure that out.

**Reply by LordNex** [reply]
https://github.com/tube0013/tube_gateways/discussions/161

**Comment by bkabela** [comment]
I can deal with two for now if it helps with the ordering process, unless you find anyone else who wants to order more and makes the numbers work.

**Comment by LordNex** [comment]
I just got a quote in for 5 @ $159.63 shipped I believe. I'm asking for a new quote for 10 now. Total cost to get them to me will Be able $32 each. Then I'll just have to split up the order and each person pay from me to them. 

Let me get the quote on and BOM for 10 and go from there. Honestly I don't think this will be that hard of a project with the right tools. I can replace the battery in the latest model iPhone. This should be a breeze hardware Wize. Hot glue does wonders

**Comment by LordNex** [comment]
Ok so here's the BOM I received for 10 boards for a total cost of $31.61 per board. I haven't tried shipping but if we're in no rush we shouldn't have to pay more than about an extra $30 to get them to the USA. Then we'll have to split them up and then person to have me ship them to them from Illinois to anywhere in the US. Let me know what you think. If I need to make the initial payment it'll take a few days to gather my resources. I only have 4 working Minis at the moment but was going to use the 5th one and see if I can find someone who doesn't want thiers or are selling a dead on on eBay. But if there's enough people that want them I'll go down to 4 for myself and leave the rest. 

So that leaves anyone who is ready to commit to this order 5-6 built boards. I am not an engineer. I did not put any of this together. I just followed this guy and I believe with Home Assistant and the Wyoming integration I should have a complete conversational AI that can control my home. I don't know if I can accomplish this and I wish I could just get 1 and see if it works. But China only does bulk. I have no idea if any of this is correct nor am I anymore than a hobby microelectronics. I make no assumptions as to your ability to build these further than me getting the board to you. Of course I think we're all trying to accomplish the same thing. Also, look up MusicAssistant. It's a perfect combo for almost any integration. So my hopes are high. 

Let me know what you think. Sorry for the images. I'm lazy and didn't feel like uploading everything. Also this is a copy of the email I received with the BoM so you have an idea. 


——

Hello,

Attached plz find the parts quotation and focus on the contents in "Notes". Some parts takes 7-10 working days --because they are imported from Digikey or Mouser to make quality 100% assured. Some parts need further information, plz check and reply one by one in the price list directly.
If any components has special requirements, plz inform me in advance. For instance, the soldering temperature needs to be controlled,  sensitive component needs to avoid water, LED needs to be firmly packaged.
The lead time for the whole order is about 22-26 days. 
Plz check design file, bom list, centroid file again to ensure all is correct before payment. You can pay directly in our system via Paypal or credit card. For bank transfer, it charges extra 30$ bank handling fee. Invoice can be download from Pcbway system to arrange payment accordingly.

Waiting for your kind early reply to proceed : )
 2023-12-01 

![image](https://github.com/justLV/onju-voice/assets/9270948/f86d0c40-d9f1-4bb5-9557-a9a76ce754a2)
![image](https://github.com/justLV/onju-voice/assets/9270948/9dc7792b-74d7-4209-bc65-6f89b9c51cb0)

**Reply by qnlbnsl** [reply]
@LordNex Let me know how to pay. I am also okay with 2 if need be. However, i will only be able to pay after the 15th (paycheck...)

**Reply by bkabela** [reply]
Just let me know how to pay, I get paid Friday and can send money then.

**Reply by LordNex** [reply]
No problem. I got to wait a few days too get the money into the right account. If you have an  phone is say  pay would be the easiest. Other than that a MO or Certified Check are fine. Most wire transfers aren't cheap so I wouldn't that. Ill try and get order put in this weekend or next week it so. 

Just let me know what works and I can send you my iPhone number.

**Reply by LordNex** [reply]
@bkabela @qnlbnsl @lifton @justLV 

This is my last call before I just either shut this down or have them reconfigure my card for just 5. I'm not sure if GitHub is blocking me or what but I haven't heard back from anyone in almost 2 weeks. I have 10 fully engineered boards ready for the green light. I have the funds to pay for them up front. You just pay the cost of the board $31 and the cost it will take to ship it to you. 

I'll leave this up for another week but after that I'm reinvesting most of my cash I took out and having them build me 5 for myself. Please reply back if you do or don't want any.

**Comment by LordNex** [comment]
Here is what they have set the price at. I still need to double check the online cart and make sure everything good. Then I'll pay the full cost to get them here, after that you can have them at cost this order, you just have to pay the cost of the boards and whatever it takes to ship it to you. 

Apple Pay is a good an inexpensive secure way to send the funds if you have a Apple Account. If not, let me know and we can work something out. If you'd like to talk about it over the phone, let me know and I'll send you my direct contact

**Reply by LordNex** [reply]
![image](https://github.com/justLV/onju-voice/assets/9270948/acb8786b-00bc-428c-ac60-0f05d2fad1dc)

**Comment by LordNex** [comment]
I give up

**Reply by droyholmes** [reply]
If you still have interest in this I’m seriously interested myself.
